exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -426,6 +445,73 @@ void getAuditorEmulatedNode() {
     }
   }
 
+  @Test
+  void timeServiceWorksInTestKit() {
+    FakeTimeProvider timeProvider = FakeTimeProvider.create(TIME);
+    try (TestKit testKit = TestKit.builder(EmulatedNodeType.VALIDATOR)
+        .withService(TestServiceModule.class)
+        .withTimeService(timeProvider)
+        .build()) {
+      // Commit two blocks for time oracle to prepare consolidated time
+      testKit.createBlock();
+      testKit.createBlock();
```

Comment:
But why two blocks (why one isn't enough)? Don't we have a single validator here?

Reply:
That's an expected behaviour. After the first block time transactions are generated and after the second one they are processed. @vitvakatu can explain in more detail if needed

Reply:
I see, thanks. Please add a comment because it is far from apparent.